src/main/java/uk/gov/companieshouse/insolvency/data/config/ApplicationConfig.java

```diff
@@ -7,25 +7,58 @@
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 import uk.gov.companieshouse.environment.EnvironmentReader;
 import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+import uk.gov.companieshouse.insolvency.data.converter.CaseDateTypeEnumReadingConverter;
 import uk.gov.companieshouse.insolvency.data.converter.CompanyInsolvencyConverter;
+import uk.gov.companieshouse.insolvency.data.converter.ModelTypeEnumReadingConverter;
+import uk.gov.companieshouse.insolvency.data.converter.RoleEnumReadingConverter;
+import uk.gov.companieshouse.insolvency.data.converter.StatusEnumReadingConverter;
 
 @Configuration
 public class ApplicationConfig implements WebMvcConfigurer {
 
     private final CompanyInsolvencyConverter insolvencyConverter;
+    private final ModelTypeEnumReadingConverter modelTypeEnumReadingConverter;
+    private final CaseDateTypeEnumReadingConverter caseDateTypeEnumReadingConverter;
+    private final RoleEnumReadingConverter roleEnumReadingConverter;
+    private final StatusEnumReadingConverter statusEnumReadingConverter;
 
-    public ApplicationConfig(CompanyInsolvencyConverter insolvencyConverter) {
+    /**
+     * Configuring converters to save and read data from mongodb.
+     * @param insolvencyConverter data type converter
+     * @param modelTypeEnumReadingConverter model type enum converter
+     * @param caseDateTypeEnumReadingConverter case data type enum converter
+     * @param roleEnumReadingConverter role enum converter
+     * @param statusEnumReadingConverter status enum converter
+     */
+    public ApplicationConfig(CompanyInsolvencyConverter insolvencyConverter,
+                             ModelTypeEnumReadingConverter modelTypeEnumReadingConverter,
+                             CaseDateTypeEnumReadingConverter caseDateTypeEnumReadingConverter,
+                             RoleEnumReadingConverter roleEnumReadingConverter,
+                             StatusEnumReadingConverter statusEnumReadingConverter) {
         this.insolvencyConverter = insolvencyConverter;
+        this.modelTypeEnumReadingConverter = modelTypeEnumReadingConverter;
+        this.caseDateTypeEnumReadingConverter = caseDateTypeEnumReadingConverter;
```

Comment:
Just wondering don't we need LocalDate and OffsetDateTime converters as well?

Reply:
The converters are implemented in separate ticket where Sameer is working.